modules/apps/headless/headless-discovery/headless-discovery-web/src/js/swagger-ui/plugins/helpInput.js

```diff
@@ -30,14 +30,18 @@ function FilterableFieldsHelp({filterableFields, messageDetails}) {
 				<>
 					<hr />
 					<ul className="c-mb-0 c-pl-0">
-						{filterableFields.map((filterableField) => (
+						{Object.keys(filterableFields).map((key) => (
 							<ClayLayout.ContentRow
 								className="c-mr-2 c-my-2"
 								containerElement="li"
-								key={filterableField}
+								key={key}
```

Comment:
I prefer continuing to use `filterableField` instead of `key` since it feels more semantic to me, but it's a personal preference. If `key` works better for you, it's ok.